polkadot/runtime/westend/src/lib.rs

```diff
@@ -1203,11 +1203,16 @@ impl pallet_message_queue::Config for Runtime {
 
 impl parachains_dmp::Config for Runtime {}
 
+parameter_types! {
+	pub const DefaultChannelSizeAndCapacityWithSystem: (u32, u32) = (4096, 4);
```

Comment:
why is this so small for westend? (rococo gets `(51200, 500)`)

Reply:
https://github.com/paritytech/polkadot-sdk/blob/212179cb990d3155463324ca59e8ed5d8ed82fc1/polkadot/node/service/src/chain_spec.rs#L144-L145

otherwise the benchmarks won't work